src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/PersonalisationProvider.java

```diff
@@ -358,6 +358,18 @@ public Map<String, String> getRespondToCostsApplicationNumber(AsylumCase asylumC
             .build();
     }
 
+    public Map<String, String> getAdditionalEvidenceApplicationNumber(AsylumCase asylumCase) {
+        DynamicList applyForCostsDynamicList = asylumCase.read(ADD_EVIDENCE_FOR_COSTS_LIST, DynamicList.class)
```

Comment:
the name of the variable probably should be addEvidenceForCostsDynamicList or smth like that